scripts/zones/PsoXja/npcs/qm_culberry.lua

```diff
@@ -12,9 +12,9 @@ local entity = {}
 
 entity.onTrade = function(player, npc, trade)
     local pendantChance = 0
-    if npcUtil.tradeHas(trade, xi.item.ODOROUS_KNIFE) then
+    if npcUtil.tradeHas(trade, xi.items.ODOROUS_KNIFE) then
         pendantChance = 500
-    elseif npcUtil.tradeHas(trade, xi.item.ODOROUS_KNIFE_1) then
+    elseif npcUtil.tradeHas(trade, xi.items.ODOROUS_KNIFE_1) then
```

Comment:
Additional scope, but could you change this to `ODOROUS_KNIFE_P1` to follow other generated naming conventions?

Reply:
Fixed it